get_tracking_numbers.py

```diff
@@ -55,7 +55,8 @@ def fill_expected_costs(all_clusters, config):
     total_trackings = sum(
         [len(trackings) for trackings in groups_dict.values()])
     print("Found %d total tracking numbers" % total_trackings)
-    email_sender.send_email(groups_dict)
+    if total_trackings > 0:
```

Comment:
let's also put the group-uploading, sheet-uploading, and writing-to-tracking file inside this if statement as well 

let's keep the cluster management outside of it just in case we need to reload the clusters file

Reply:
Done.